src/App.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import { useState } from 'react';
 
 const App = () => {
+  const [likesCount, setLikesCount] = useState(0);
```

Comment:
Good use of state to keep track of the likesCount, however, you will also need a state variable to track the changes to our messages.